### PR TITLE
Adapt BIO_new_accept() to call BIO_set_accept_name()

### PR DESCRIPTION
**Pull request by jamercee**
Commit 417be66 broke BIO_new_accept() by changing the definition of the
macro BIO_set_accept_port() which stopped acpt_ctrl() from calling
BIO_parse_hostserv(). This commit completes the series of changes
initiated in 417be66.

Updated pods to reflect new definition introduced by 417be66.

### CODE REVIEW

**Comment by kaduk** [comment]
This leaves BIO_set_accept_port() undocumented as to its actual functionality by changing references from _port to _name -- both should be documented.

**Comment by jamercee** [comment]
I'll take care of it.

File: doc/crypto/BIO_s_accept.pod [hunk 1]
```diff
@@ -134,17 +143,20 @@ then it is an indication that an accept attempt would block: the application
...
 =head1 RETURN VALUES
 
 BIO_do_accept(),
-BIO_set_accept_port(), BIO_set_nbio_accept(), BIO_set_accept_bios(),
-and BIO_set_bind_mode(), return 1 for success and 0 or -1 for failure.
+BIO_set_accept_name(), BIO_set_accept_port(), BIO_set_nbio_accept(),
+BIO_set_accept_bios(), and BIO_set_bind_mode(), return 1 for success and 0 or
+-1 for failure.
+
+BIO_set_accept_name() returns the accept name or NULL on error.
```
Comment (kaduk) [inline doc/crypto/BIO_s_accept.pod]: s/set/get/

**Comment by kroeckx** [comment]
+1

**Comment by jamercee** [comment]
Will this be merged?

**Comment by richsalz** [comment]
yes, since it has the 1.1.0 milestone

**Comment by richsalz** [comment]
I pushed this, at commit 5898b8e.  Thanks!

**Comment by jamercee** [comment]
Hi Rich,

 

This commit does not seem related to PR #1386, which should have modified crypto/bios/bss_acpt.c

 

According to https://github.com/openssl/openssl/pull/1386#event-751361130:

 

This pull request is closed, but the jamercee:fix-BIO_new_accept branch has unmerged commits.

**Comment by richsalz** [comment]
sorry, commented on wrong MR.  Now really pushed.

**Pull request closed without merging**